app/controllers/commodities_controller.rb

```diff
@@ -29,7 +29,17 @@ def declarable
   end
 
   def uk_declarable
-    @uk_declarable ||= heading? ? uk_heading : uk_commodity
+    @uk_declarable ||= begin
+      heading? ? uk_heading : uk_commodity
+    rescue Faraday::ResourceNotFound
+      if TradeTariffFrontend::ServiceChooser.xi?
+        # When loading EU commodities we pull some measures from the UK commodity. If the UK commodity is not present
+        # in the UK database, then we should show only the applicable EU measures.
+        nil
+      else
+        raise
+      end
```

Comment:
this looks the same as:
`raise if TradeTariffFrontend::ServiceChooser.xi?`

Reply:
Derp, yeah